### ISSUE TITLE
[error]

### ISSUE THREAD

**Issue by mmgil** [olljanat/portainer-vsts-deploy#2]
I Receive following error in Azure Devops using ubuntu-16.04 how agent for my pipeline:

*[error]The current operating system is not capable of running this task. That typically means the task was written for Windows only. For example, written for Windows Desktop PowerShell.*

**Comment by olljanat** [comment]
Yes that is (or at least was on time when I created this one) unfortunate limitation on Azure DevOps that custom extensions are using old PowerShell instead PowerShell Core which would works on Linux too.
